README.md

```diff
@@ -46,14 +46,23 @@ bevy_ios_safearea = { version = "=0.1.0" }
 
 Initialize Bevy Plugin:
 
-```rust
+```rust,norun
 // request initialisation right on startup
-app.add_plugins(IosSafeAreaPlugin);
+use bevy::prelude::*;
+App::new()
+    .add_plugins((DefaultPlugins,bevy_ios_safearea::SafeAreaPlugin));
 ```
 
+Accessing it:
+
 ```rust
-fn bevy_system(safe_area: Res<IosSafeAreaResource>) {
-    let safe_area_top = safe_area.top();
+use bevy::prelude::*;
+fn bevy_system(safe_area: Option<Res<bevy_ios_safearea::IosSafeArea>>) {
+    let Some(area) = safe_area else {
```

Comment:
i would still keep the usage example of `top()` around which defaults to 0 on non-ios

Reply:
as like a convenience method

Reply:
Restored with a trait ;)